exercises/prime-factors/example.jl

```diff
@@ -0,0 +1,34 @@
+function isPrime(n)
```

Comment:
In Julia, function names are usually all lowercase, sometimes with `_` as word separators (snakecase). This function should follow that convention and be renamed to `isprime`